### PR TITLE
Message link in User view

### PR DESCRIPTION
**Pull request by xuwhite**
New feature: from the user view you can now click the envelope icon and get to your messages with that user.

### CODE REVIEW

File: includes/view/User_view.php [hunk 1]
```diff
@@ -677,11 +677,18 @@ function User_view(
             ]),
             div('row user-info', [
                 div('col-md-2', [
-                    heading(icon('phone')
-                        . '<a href="tel:' . $user_source->contact->dect . '">'
-                        . $user_source->contact->dect, 1)
-                        . '</a>'
-                ]),
+                    heading(
```
Comment (MyIgel) [inline includes/view/User_view.php:680]: Eventually a check if `enable_dect` is configured would be nice here

File: includes/view/User_view.php [hunk 2]
```diff
@@ -677,11 +677,18 @@ function User_view(
...
+                        icon('phone')
+                        . ' <a href="tel:' . $user_source->contact->dect . '">'
+                        . $user_source->contact->dect
+                        . '</a>', 1),
+                    heading(
```
Comment (MyIgel) [inline includes/view/User_view.php:685]: And a check for `user_messages` there would be great

Comment (xuwhite) [reply includes/view/User_view.php]: done

File: includes/view/User_view.php (hunk 1 above)
Comment (xuwhite) [reply includes/view/User_view.php]: done

File: includes/view/User_view.php [hunk 3]
```diff
@@ -677,11 +677,24 @@ function User_view(
...
-                ]),
+                    if (config('enable_dect')) {
+                        heading(
+                            icon('phone')
+                            . ' <a href="tel:' . $user_source->contact->dect . '">'
+                            . $user_source->contact->dect
+                            . '</a>', 1
+                        ),
+                    }
+                    if (config('enable_dect')) {
```
Comment (MyIgel) [inline includes/view/User_view.php:688]: I don't think so?

File: includes/view/User_view.php [hunk 4]
```diff
@@ -677,11 +677,24 @@ function User_view(
             ]),
             div('row user-info', [
                 div('col-md-2', [
-                    heading(icon('phone')
-                        . '<a href="tel:' . $user_source->contact->dect . '">'
-                        . $user_source->contact->dect, 1)
-                        . '</a>'
-                ]),
+                    if (config('enable_dect')) {
```
Comment (MyIgel) [inline includes/view/User_view.php:680]:
```
syntax error, unexpected token "if", expecting "]"
 ParseError
…/includes/view/User_view.php680 
```

**Review by MyIgel — CHANGES_REQUESTED** [review]

File: includes/view/User_view.php (hunk 3 above)
Comment (xuwhite) [reply includes/view/User_view.php]: 😖

File: includes/view/User_view.php (hunk 4 above)
Comment (xuwhite) [reply includes/view/User_view.php]: 😖

**Comment by MyIgel** [comment]
Danke für den PR! :heart_eyes:

**Pull request merged**